lib/logstash/outputs/email.rb

```diff
@@ -125,7 +125,7 @@ def register
   def receive(event)
 
 
-      @logger.debug? and @logger.debug("Creating mail with these settings : ", :via => @via, :options => @options, :from => @from, :to => @to, :cc => @cc, :subject => @subject, :body => @body, :content_type => @contenttype, :htmlbody => @htmlbody, :attachments => @attachments, :to => to, :to => to)
+      @logger.debug? and @logger.debug("Creating mail with these settings : ", :via => @via, :options => @options, :from => @from, :to => @to, :cc => @cc, :subject => @subject, :body => @body, :content_type => @contenttype, :htmlbody => @htmlbody, :attachments => @attachments)
```

Comment:
shouldn't the proper fix to just remove the duplicated `:to => to` instead of removing the two `:to => to, :to => to` ?

Reply:
On the same line but at the beginning there is: `:to => @to`.